src/04-modality-specific-files/04-intracranial-electroencephalography.md

```diff
@@ -143,10 +143,10 @@ Specific iEEG fields SHOULD be present:
-| iEEGPlacementScheme             | RECOMMENDED           | [string][]                           | Freeform description of the placement of the iEEG electrodes. Left/right/bilateral/depth/surface (for example, "left frontal grid and bilateral hippocampal depth" or "surface strip and STN depth" or "clinical indication bitemporal, bilateral temporal strips and left grid").                                                     |
-| iEEGElectrodeGroups             | RECOMMENDED           | [string][]                           | Field to describe the way electrodes are grouped into strips, grids or depth probes for example, {'grid1': "10x8 grid on left temporal pole", 'strip2': "1x8 electrode strip on xxx"}.                                                                                                                                                 |
-| SubjectArtefactDescription      | RECOMMENDED           | [string][]                           | Freeform description of the observed subject artefact and its possible cause (for example, "door open", "nurse walked into room at 2 min", "seizure at 10 min"). If this field is left empty, it will be interpreted as absence of artifacts.                                                                                          |
+| iEEGGround                      | RECOMMENDED           | [string][]                           | Description of the location of the ground electrode (`"placed on right mastoid (M2)"`).                                                                                                                                                                                                                                                  |
+| iEEGPlacementScheme             | RECOMMENDED           | [string][]                           | Freeform description of the placement of the iEEG electrodes. Left/right/bilateral/depth/surface (for example, `"left frontal grid and bilateral hippocampal depth"` or `"surface strip and STN depth"` or `"clinical indication bitemporal, bilateral temporal strips and left grid"`).                                                     |
+| iEEGElectrodeGroups             | RECOMMENDED           | [string][]                           | Field to describe the way electrodes are grouped into strips, grids or depth probes for example, `"{'grid1': "10x8 grid on left temporal pole", 'strip2': "1x8 electrode strip on xxx"}"`.                                                                                                                                                 |
```

Comment:
just saw that this actually addresses https://github.com/bids-standard/bids-specification/issues/686

So let's make it absolutely clear that this is a string, not an object (or "string of an object").

Dora's description here is convincing IMHO: https://github.com/bids-standard/bids-specification/issues/686#issuecomment-737333966

```suggestion
| iEEGElectrodeGroups             | RECOMMENDED           | [string][]                           | Field to describe the way electrodes are grouped into strips, grids or depth probes for example, `"grid1: 10x8 grid on left temporal pole, strip2: 1x8 electrode strip on xxx"`.                                                                                                                                                 |
```

Reply:
Good with me. :-)